cmd/kubelet/app/options/options.go

```diff
@@ -105,7 +105,6 @@ type KubeletFlags struct {
 	// The Kubelet will load its initial configuration from this file.
 	// The path may be absolute or relative; relative paths are under the Kubelet's current working directory.
 	// Omit this flag to use the combination of built-in default configuration values and flags.
-	// To use this flag, the KubeletConfigFile feature gate must be enabled.
 	KubeletConfigFile flag.StringFlag
```

Comment:
can just be a string, and check len(KubeletConfigFile) > 0 instead of `Provided()`

Reply:
sgtm

Reply:
note to self: we should do the same for dynamic config dir, but in a separate PR

Reply:
done